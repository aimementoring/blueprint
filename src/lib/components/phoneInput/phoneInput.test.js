import React from 'react';
import renderer from 'react-test-renderer';
import PhoneInput from './phoneInput.js';

describe('PhoneInput', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(<PhoneInput currentSite="au" placeholder="Phone" name="phone" value="" required />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
