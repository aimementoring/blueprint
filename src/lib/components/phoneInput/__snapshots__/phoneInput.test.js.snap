// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoneInput renders properly 1`] = `
<div
  className="theme-base"
>
  <div
    className=" inputWrapper"
  >
    <div
      className="intl-tel-input"
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <div
        className="flag-container"
      >
        <div
          className="selected-flag"
          onClick={[Function]}
          onKeyDown={[Function]}
          tabIndex=""
          title="undefined: +undefined"
        >
          <div
            className="iti-flag us"
          />
          <div
            className="selected-dial-code"
          >
            
          </div>
          
        </div>
        <ul
          className="country-list hide"
        />
      </div>
      <input
        autoComplete="off"
        autoFocus={false}
        className="input"
        disabled={false}
        id=""
        name="phone"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onPaste={[Function]}
        placeholder="Phone"
        readOnly={false}
        required={true}
        type="tel"
        value=""
      />
    </div>
  </div>
</div>
`;
